### DISCUSSION TITLE
Search Filter WebPart | AAD Properties

### DISCUSSION THREAD

**Discussion by Gryphos87** [microsoft-search/pnp-modern-search#2434] [Q&A]
Hello,

Is it possible to make use of AAD properties in the Search Filter WebPart? We use the SharePoint Search Data source in the Search Results WebPart. The context is to create a people search application.

Cheers

**Comment by wobba** [comment]
People search via the SharePoint provider uses SharePoint user profile data. So you need to sync AAD properties to UPA for that scenario to work.

**Reply by Gryphos87** [reply]
Hello @wobba,

Is there any other way get AAD properties in the Search Filter? Maybe by switching the data source to "Microsoft Search"? Can I still build an user search application with this data source then?

Cheers

**Reply by wobba** [reply]
Not to my knowledge. Refer to the Graph documentation on capabilities offered there. If you build something from scratch you can build whatever you want on top of public API's :)
